### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by Eccb7**
## Description
This pull request addresses the following project requirements and enhancements:

## Project Requirements
Added a new column species of type string to the animals table.
- Inserted additional data for various animals into the animals table.
- Executed transactions and queries as specified in the project requirements.
- Updated the data.sql and queries.sql files to reflect the changes.
- Provided screenshots for the results of queries and transactions.

## Screenshots
Screenshot from 2023-09-01 09-01-52

Screenshot from 2023-09-01 11-20-43

Screenshot from 2023-09-01 11-26-12

Screenshot from 2023-09-01 11-28-25

Screenshot from 2023-09-01 11-29-58

Screenshot from 2023-09-01 11-30-39

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -13,3 +13,41 @@ SELECT * FROM animals WHERE neutered = true;
...
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+-- Transaction: Update species based on conditions
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
```
Comment (bienvenuushindi) [inline queries.sql]: :red_circle: Before you commit the transaction. The [milestone requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md) state that you should verify that changes were made. :arrow_down: . Kindly make sure you include the query for this requirement.

CHANGE

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
Hi @Eccb7 , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the assessment but you are almost there!



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## Suggested changes

- [ ] :red_circle: :red_circle:  **Kindly add screenshots of your query and transaction results to this pull request description. This will help anyone reviewing your project to check if your results are correct. This also helps to ensure that you run the queries with success.** :red_circle: :red_circle: 
## :stop_sign:  NOTE: Kindly Edit this Pull Request Description to add the screenshots. Please avoid creating a new Pull Request 
- [ ]  _Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: queries.sql (hunk 1 above)
Comment (Eccb7) [reply queries.sql]: Hi @bienvenuushindi 
Thanks for the update.

Comment (V-Blaze) [inline queries.sql]: - [ ] As pointed out by the previous reviewer, and stated in the project requirement, you need to verify the changes before you commit. :+1: :smile: :small_airplane:  You need to update your query to this effect. here's a guide:
```sh
-- Transaction: Update species based on conditions

BEGIN;
UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
UPDATE animals SET species = 'pokemon' WHERE species IS NULL;

-- Verify the changes
SELECT * FROM animals;

COMMIT;
SELECT * FROM animals;
```

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @Eccb7 

# Almost there 🟡 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, and some changes from the previous reviewers, but there are still some issues I think you need to improve on. 😄 👋 


## Highlights
- Schema is written correctly :heavy_check_mark: 
- Documented code :heavy_check_mark: 
- Correct Git flow :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi @Eccb7 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Good work updating your query history to confirm the changes 👍 
✅ You did a good job documenting the query results👍 


## Optional suggestions

N/A



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**